content/en/_index.md

```diff
@@ -11,22 +11,12 @@ developer_note:
 
 ## High-quality, ubiquitous, and portable telemetry to enable effective observability
 
-<div class="l-primary-buttons mt-5">
 
-- [Learn more](/docs/concepts/)
-- [Mission, vision, and values](/mission/)
+<div class="h3 mt-4">Start your OpenTelemetry journey</div>
+<div class="l-primary-buttons">
 
-</div>
-
-<div class="h3 mt-4">Get started!</div>
-<div class="l-get-started-buttons">
-
-- [Collector](/docs/collector/getting-started/)
-- [Java](/docs/instrumentation/java/getting-started/)
-- [Go](/docs/instrumentation/go/getting-started/)
-- [.NET](/docs/instrumentation/net/getting-started/)
-- [JavaScript](/docs/instrumentation/js/getting-started/)
-- <a href="/docs/instrumentation/" title="All languages"><i class="fas fa-ellipsis-h"></i></a>
+- [Dev](/docs/getting-started/dev)
+- [Ops](/docs/getting-started/ops)
```

Comment:
I'm not sure how it could be achieved, but I'd propose that we keep the **Learn more** button, who just want to learn more without having to be pigeonholed into **Dev** or **Op** -- which might just be confusing for some. WDYT?

Reply:
yes, I think that's true... I was thinking about having a "Learn More" or "See OpenTelemetry in Action" thingy where people get a super quick intro and then can watch a video of the demo and try things out.